scripts/CMakeLists.txt

```diff
@@ -124,6 +124,12 @@ STRING(REPLACE       "-fstack-protector-strong"      "" CXXFLAGS "${CXXFLAGS}")
 STRING(REPLACE       "-grecord-gcc-switches"         "" CXXFLAGS "${CXXFLAGS}")
 STRING(REPLACE       "-pipe"                         "" CXXFLAGS "${CXXFLAGS}")
 
+IF(CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
+  SET (PERL_PATH "/usr/local/bin/perl")
```

Comment:
Our scripts/mysqldumpslow.sh also falls under the the s/#!/usr/bin/perl/@PERL_PATH@ /g replacement (I have no idea why upstream only changed scripts dir when there are Perl scripts everywhere)